Demo/src/main/java/com/paypal/android/ui/paypal/PayPalFragment.kt

```diff
@@ -1,26 +1,75 @@
 package com.paypal.android.ui.paypal
 
+import android.os.Build
 import android.os.Bundle
 import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Button
 import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.fragment.app.viewModels
+import com.paypal.android.BuildConfig
+import com.paypal.android.R
+import com.paypal.android.checkout.PayPalClient
+import com.paypal.android.core.CoreConfig
+import com.paypal.android.core.Environment
 import com.paypal.android.ui.theme.DemoTheme
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
 class PayPalFragment : Fragment() {
+
+    private val payPalViewModel: PayPalViewModel by viewModels()
+
+    private val canRunPayPalCheckout = Build.VERSION.SDK_INT >= Build.VERSION_CODES.M
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
+
+        val coreConfig = CoreConfig(BuildConfig.CLIENT_ID, environment = Environment.SANDBOX)
+        val application = requireActivity().application
+        val returnUrl = BuildConfig.APPLICATION_ID + "://paypalpay"
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
Looks like we can use the `canRunPayPalCheckout` val here.

Reply:
@tdchow I tried, but the compiler and linter throw me errors/warnings, because they cant detect that im actually checking for api version.

Reply:
Ah, that rule is probably expecting to see the actual SDK level check.